docs/use-mainnet/gas-on-linea.mdx

```diff
@@ -6,42 +6,94 @@ image: /img/socialCards/gas-on-linea.jpg
 
 ## What does gas pay for?
 
-Gas must be paid to cover any kind of on-chain computation. For a zk rollup like Linea, this mainly involves execution and storage on L2, and publishing `calldata` on L1.
+Gas must be paid to cover any kind of on-chain computation. For a zk rollup like Linea, this mainly
+involves execution and storage on L2 and posting L2 transaction and messaging data to L1.
 
-The main costs of interacting with L1 are:
-- Verifying 'proofs', the cryptographic functions that confirm the validity of the state changes being recorded on L1, and; 
-- Publishing compressed transactions to L1 as `calldata`.
+Of these, interacting with L1 has historically been far more costly. The main costs associated 
+with L1 are: 
 
-Since the process of publishing data and verifying zk proofs on Ethereum mainnet is an indispensable and vital feature of Linea, we can consider the Linea gas price to cover two dimensions: the L2 execution and storage resources on Linea, and the L1 cost of guaranteeing the computational integrity of each Linea batch.
+- Verifying 'proofs', the cryptographic functions that confirm the validity of the state changes 
+being recorded on L1. 
+- Posting data about L2 transactions in 'blobs', although this [costs far less than the method it 
+replaced](#eip-4844-proto-danksharding). 
 
-## Alpha v2 and minimizing gas fees
+Since the process of publishing data and verifying zk proofs on Ethereum mainnet is an 
+indispensable and vital feature of Linea, we can consider the Linea gas price to cover two 
+dimensions: the L2 execution and storage resources on Linea, and the L1 cost of guaranteeing 
+the computational integrity of each Linea batch.
 
-The Alpha v2 upgrade came to Linea mainnet on February 13, and made several improvements that have reduced gas fees by approximately 66%.
+## Minimizing gas fees
 
-### Data compression
+Since the network's inception, offering cheap, accessible transaction costs has been a core 
+objective for Linea. 
 
-Data compression significantly reduces the cost of posting `calldata`—L2 transaction data and messaging data—to L1.
+Iterative changes have gradually driven these costs down; for example, the Alpha v2 upgrade in 
+February 2024 introduced an improved data compression algorithm that dramatically reduced L1 
+computational costs by enabling more transactions to be included in each batch submitted to L1. 
 
-This optimization takes advantage of the Linea prover's existing ability to operate with a compressed input, decompressing `calldata` to validate L2 state transitions. The `calldata` passed to the L1 smart contract can therefore be considerably smaller, driving down the sum cost of transactions in each block.
+Linea's latest mainnet upgrade, Alpha v3, builds on this progress by implementing EIP-4844, or 
```

Comment:
You may want to avoid using "latest"here. When the next upgrade happens this will no longer be the case.